packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
```

Comment:
@llorca as a _very_ broad/rough rule, I would prefer custom classes because they reduce CSS specificity

@greglo I think the name looks good